lib/methods/export.js

```diff
@@ -133,7 +170,7 @@ module.exports = self => {
             $in: publishedIds
           }
         })
-        .relationships(includeRelationships)
+        .relationships(false)
```

Comment:
unneeded since we get related docs manually

Reply:
Avoiding the redundant call should be a nice little perf win then.